recipes/rb-dotenv/meta.yaml

```diff
@@ -0,0 +1,44 @@
+{% set name = "dotenv" %}
+{% set version = "2.7.5" %}
+
+package:
+  name: rb-{{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://rubygems.org/downloads/{{ name }}-{{ version }}.gem
+  sha256: c5465769f188f96a62a6cc1f801f440af077b06b788a14f66760a5178569139e
+
+build:
+  noarch: generic
+  number: 0
+  script:
+    - gem install --norc -l -V --ignore-dependencies {{ name }}-{{ version }}.gem
+    - gem unpack {{ name }}-{{ version }}.gem
+  skip: {{ win }}
+
+requirements:
+  host:
+    - ruby
+  run:
+    - ruby
+
+test:
+  commands:
+    - ruby -r {{ name }} -e 'exit 0'
+    - dotenv --help
+
+about:
+  home: https://rubygems.org/gems/{{ name }}
+  license: MIT
+  license_family: MIT
+  license_file: {{ name }}-{{ version }}/LICENSE
+  summary: 'Ruby gem to load environment variables from `.env`'
+  doc_url: https://www.rubydoc.info/gems/dotenv/2.7.5
+  dev_url: https://github.com/bkeepers/dotenv
+
+extra:
+  recipe-maintainers:
+    - rluria14
```

Comment:
You maybe want to add @sodre who is currently spearheading the ruby package builds.